src/ts/components/account/NetworkCard.tsx

```diff
@@ -0,0 +1,99 @@
+import * as React from 'react'
+import styled from 'styled-components'
+
+interface INetworkCardProps {
+  imgPath: string
+  name: string
+  supported: boolean
+}
+
+interface INetworkCardState {
+  selected: boolean
+}
+
+export default class NetworkCard extends React.Component<INetworkCardProps, INetworkCardState> {
+
+  state = {
+    selected: false
+  }
+
+  render () {
+    return this.state.selected ? this.renderClicked() : this.renderDefault()
+  }
+
+  renderDefault () {
+    return (
+      <Card onClick={this.toggleSelected}>
+        <NetworkDetail>
+          <NetworkImage src={this.props.imgPath} alt={'chain-logo'}/>
+          <div>
+            <NetworkName>
+              {this.props.name}<br/><span>{!this.props.supported ? '(comming soon)' : ''}</span>
+            </NetworkName>
+          </div>
+        </NetworkDetail>
+      </Card>
+    )
+  }
+
+  renderClicked () {
+    return (
+      <Card onClick={this.toggleSelected}>
+        <NetworkDetail>
+          <NetworkImage src={this.props.imgPath} alt={'chain-logo'}/>
+          <div>
```

Comment:
Do we need this `div`

Reply:
From property ‘justify-content: center’ for text in the card, it is needed.

Reply:
Not anymore